test/demarcheSimplifiees/autoVerifyPsychologist.spec.js

```diff
@@ -13,16 +13,17 @@ describe('autoVerifyPsychologist', () => {
     unsets = [];
     executeMutationStub = sinon.stub();
     unsets.push(graphql.__set__('executeMutation', executeMutationStub));
-    unsets.push(autoVerifyPsychologists.__set__('graphql', graphql));
+    autoVerifyPsychologists.__Rewire__('graphql', graphql);
   });
 
   afterEach((done) => {
+    autoVerifyPsychologists.__ResetDependency__('graphql');
     unsets.forEach((unset) => unset());
     done();
   });
 
   it('Should execute 4 mutations on DS', async () => {
-    await autoVerifyPsychologists();
+    await autoVerifyPsychologists.default();
```

Comment:
What does this `.default()` do?

Reply:
because i'm importing the all modules to rewire it (l16) so here I need to call the default exported method of the module